.circleci/config.yml

```diff
@@ -277,6 +277,18 @@ jobs:
       - *setmetapackageversion
       - *builddebianpackage
 
+  build-nightly-buster-securedrop-workstation-svs-disp:
+    docker:
+      - image: circleci/python:3.7-buster
+    steps:
+      - checkout
+      - *installdeps
+      - *setsvsdispname
+      - *setmetapackageversion
```

Comment:
does this bump the version? i think with this it will just keep committing versions of the same package

Reply:
Agreed. At a glance, we don't have sufficient logic to increment here, we'll need to write something new. Neither `&getlatestreleasedversion` nor `&getnightlyversion` are what we want, because both assume the package has a tag that matches its version, which isn't true since this package is managed inside this repo.